src/vs/workbench/api/common/extHostTypes.ts

```diff
@@ -1227,6 +1227,25 @@ export class DocumentHighlight {
 	}
 }
 
+@es5ClassCompat
+export class MultiDocumentHighlight {
+
+	uri: URI;
+	highlights: DocumentHighlight[];
+
+	constructor(uri: UriComponents, highlights: DocumentHighlight[]) {
```

Comment:
`UriComponents` shouldn't be spec here. We expect a true `Uri` instance like in `Location`